book/src/dev/zebra-dependencies-for-audit.md

```diff
@@ -0,0 +1,150 @@
+| [tokio](https://github.com/tokio-rs/tokio) | 1.21.2 |
+| [tokio-util](https://github.com/tokio-rs/tokio) | 0.7.4 | 
+| [tower](https://github.com/tower-rs/tower) | 0.4.13 |
+| [futures-util](https://github.com/rust-lang/futures-rs) | [0.3.24](https://github.com/rust-lang/futures-rs/releases/tag/0.3.24) |
+| [tokio-stream](https://github.com/tokio-rs/tokio) | 0.1.10 |
+| [hyper](https://github.com/hyperium/hyper) | [0.14.20](https://github.com/hyperium/hyper/releases/tag/v0.14.20) |
+| [jsonrpc-core](https://github.com/paritytech/jsonrpc) | [18.0.0](https://github.com/paritytech/jsonrpc/releases/tag/v18.0.0) | 
+| jsonrpc-derive | 18.0.0 
+| [jsonrpc-http-server](https://github.com/paritytech/jsonrpc) | [18.0.0](https://github.com/paritytech/jsonrpc/releases/tag/v18.0.0) | 
+
+## Types and encoding
+
+| Name | Version | Notes
+|------| ------- | -----
+| [bitflags](https://github.com/bitflags/bitflags) | [1.3.2](https://github.com/bitflags/bitflags/releases/tag/1.3.2) 
+| [bitvec](https://github.com/bitvecto-rs/bitvec) | 1.0.1 |  We use it to build bit vectors, which are used when computing commitments. It's important, but does not seem particularly risky.
+| [byteorder](https://github.com/BurntSushi/byteorder) | [1.4.3](https://github.com/BurntSushi/byteorder/releases/tag/1.4.3) 
+| [chrono](https://github.com/chronotope/chrono) | [0.4.22](https://github.com/chronotope/chrono/releases/tag/v0.4.22) | We treat chrono as a time library, and assume it works. It only implements the consensus rule about the local clock.
+| [hex](https://github.com/KokaKiwi/rust-hex) | [0.4.3](https://github.com/KokaKiwi/rust-hex/releases/tag/v0.4.3) 
+| [humantime](https://github.com/tailhook/humantime) | [2.1.0](https://github.com/tailhook/humantime/releases/tag/v2.1.0) 
+| [itertools](https://github.com/rust-itertools/itertools) | 0.10.5 
+| [serde](https://github.com/serde-rs/serde) | [1.0.145](https://github.com/serde-rs/serde/releases/tag/v1.0.145) 
+| [serde-big-array](https://github.com/est31/serde-big-array) | [0.4.1](https://github.com/est31/serde-big-array/releases/tag/v0.4.1) 
+| [serde_with](https://github.com/jonasbb/serde_with) | [2.0.1](https://github.com/jonasbb/serde_with/releases/tag/v2.0.1) 
+| [uint](https://github.com/paritytech/parity-common) | 0.9.4 
+| [bytes](https://github.com/tokio-rs/bytes) | [1.2.1](https://github.com/tokio-rs/bytes/releases/tag/v1.2.1) 
+| [humantime-serde](https://github.com/jean-airoldie/humantime-serde) | 1.1.1 
+| [indexmap](https://github.com/bluss/indexmap) | [1.9.1](https://github.com/bluss/indexmap/releases/tag/1.9.1) 
+| [ordered-map](https://github.com/qwfy/ordered-map.git) | 0.4.2 
+| [serde_json](https://github.com/serde-rs/json) | [1.0.85](https://github.com/serde-rs/json/releases/tag/v1.0.85) 
+| [bincode](https://github.com/servo/bincode) | [1.3.3](https://github.com/servo/bincode/releases/tag/v1.3.3) 
+| [mset](https://github.com/lonnen/mset) | [0.1.0](https://github.com/lonnen/mset/releases/tag/0.1.0)  
```

Comment:
This crate is used to store `sprout_anchors`, `sapling_anchors` and `orchard_anchors` in the non finalized state. It is also used to store transaction ids in the state for the `getaddresstxids` rpc call. The crate has no dependents: https://crates.io/crates/mset/reverse_dependencies

Reply:
We want to avoid auditing code we didn't write.